PRD.txt

```diff
@@ -99,7 +99,7 @@ For desktop use cases the CoreOS group encourages users to leverage the Fedora S
 The Fedora CoreOS working group will deliver a Fedora CoreOS
 distribution that:
 
-* automatically updates on a regular cadence, with occasional out-of-cycle security updates
+* automatically updates on a regular cadence, with occasional out-of-cycle security or critical bug fix updates
```

Comment:
I'd be okay dropping `critical`.  We don't need to promise not to fix lesser bugs.